Lib/asyncio/tasks.py

```diff
@@ -578,14 +578,13 @@ def __sleep0():
     yield
 
 
-async def sleep(delay, result=None, *, loop=None):
+async def sleep(delay, result=None):
```

Comment:
I think that just removing the loop argument may be bad for compatibility reasons. Perhaps it is better to catch the argument and raise an DeprecationWarning, as proposed in the issue

Reply:
@fbidu very good. I will change for this and send the PR again. Thank you for your review.

Reply:
Yes, @fbidu is right.